pilot/cmd/pilot-agent/options/options.go

```diff
@@ -104,6 +104,18 @@ var (
 	wasmInsecureRegistries = env.RegisterStringVar("WASM_INSECURE_REGISTRIES", "",
 		"allow agent pull wasm plugin from insecure registries or https server, for example: 'localhost:5000,docker-registry:5000'").Get()
 
+	wasmModuleExpiry = env.RegisterDurationVar("WASM_MODULE_EXPIRY", 24*time.Hour,
+		"cache expiration duration for a wasm module.").Get()
+
+	wasmPurgeInterval = env.RegisterDurationVar("WASM_PURGE_INTERVAL", 10*time.Minute,
+		"interval between checking the expiration of wasm modules").Get()
+
+	wasmHTTPRequestTimeout = env.RegisterDurationVar("WASM_HTTP_REQUEST_TIMEOUT", 5*time.Second,
```

Comment:
5s looks too short, it is downloading a module.   Change it to 15s

Reply:
Done